mono/sgen/sgen-pointer-queue.c

```diff
@@ -39,7 +39,7 @@ realloc_queue (SgenPointerQueue *queue)
 	sgen_free_internal_dynamic (queue->data, sizeof (void*) * queue->size, queue->mem_type);
 	queue->data = new_data;
 	queue->size = new_size;
-	SGEN_LOG (4, "Reallocated pointer queue to size: %lu", new_size);
+	SGEN_LOG (4, "Reallocated pointer queue to size: %zu", new_size);
```

Comment:
%z is not portable to msvc runtime.

Reply:
This is portable for msvc versions we support VS2015+ and its VS2015 triggering warning around use of %lu for size_t so fine there, but it still has issues on mingw. Since original is %lu it should be OK to type cast to unsigned long to not bring up additional warnings on mingw.